docs/root/intro/version_history.rst

```diff
@@ -6,6 +6,7 @@ Version history
 
 * http : added idle timeout for :ref:`upstream http connections
   <envoy_api_field_core.HttpProtocolOptions.idle_timeout>`.
+* lua http filter: extended to support :ref:`metadata object <config_http_filters_lua_metadata_wrapper>` API.
```

Comment:
Can you merge master? Also, please make sure this entry is alpha sorted in the final list. Thanks!